pkg/integration/enx_redirect_test.go

```diff
@@ -42,9 +42,14 @@ func TestENXRedirect(t *testing.T) {
 
 	realm := bs.Realm
 	realm.EnableENExpress = true
-	realm.SMSTextTemplate = "[enslink]"
+	template := "[enslink]"
+	realm.SMSTextTemplate = template
+	for k := range realm.SMSTextAlternateTemplates {
+		realm.SMSTextAlternateTemplates[k] = &template
+	}
+
 	if err := server.Database.SaveRealm(realm, database.SystemTest); err != nil {
-		t.Fatal(err)
+		t.Fatalf("err: %v, errors: %+v", err, realm.ErrorMessages())
```

Comment:
Can we add more content to this message if we're going to customize it?

Reply:
I put this back, this was debugging.